tests/integration/steps/data_steps.py

```diff
@@ -34,3 +36,27 @@ def dynamodb_check_delete(context, id):
         response["ResponseMetadata"]["HTTPHeaders"]["content-length"]
     ).is_equal_to("2")
     assert_that(response).does_not_contain("Item")
+
+
+@given(
+    "I have the following csv file {file_name}, I can retrieve the data within the file"
+)
+def read_csv_data(context, file_name):
+    # Open the CSV file in read mode
+    filename = read_config("csv_files", file_name)
+    file_name = os.path.basename(filename)
+    with open(filename, "r") as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=",")  # Specify the delimiter (usually a comma)
+        headers = next(csv_reader)
+        column_count = len(headers)
+        # Convert csv_reader to a list so we can iterate over it multiple times
```

Comment:
Can there be separate steps that check for the correctness of these items?
For example: the file <insert name> has the correct data <insert value> in row <insert value> of column <insert value>
the file <insert name> has the correct row count <insert value>

Reply:
All added. Please review